Jarvis-Platform/jarvis-studio
# Feature/other runs improvements

[comment]: [![Status](https://img.shields.io/badge/Status-READY-brightgreen.svg)]()
[comment]: [![Status](https://img.shields.io/badge/Status-HOLD-inactive.svg)]()

[![Status](https://img.shields.io/badge/Status-DEVELOPMENT-orange.svg)]()


## Description
Update listing in `Other runs` tab.


## Todos
- [x] Extract from current listing views (for runs) the listing component + related properties as show below to create new dedicated components:
```javascript
<listing-component
    :type="listingType"
    :module-name="moduleName"
    :headers="headers"
    :overridden-columns="overriddenColumns"
    sort-by="dag_execution_date"
    :sort-desc="true"
    show-airflow-action
>
...
</listing-component>
```
- [x] Import them in other runs tab to show the good one
- [ ] Update filters & other properties that needs to be updated in tab


## Deploy Notes
Review & merge


## Steps to Test or Reproduce
```sh
git checkout master
git pull
git checkout feature/other-runs-improvements
```

1. Go to a run `ItemView`
2. Go into `other-runs`tab & see new table with same fields as in `ListingView`


## Impacted Areas in Application
List general components of the application that this PR will affect:
- `data-workflows`
